### DISCUSSION TITLE
Ability to Hide the Lock button only

### DISCUSSION THREAD

**Discussion by Phixius5968** [homebridge-plugins/homebridge-august#24] [Ideas]
Hi there, 
Since I have the lock added to HomeKit natively, adding it via homebridge-august double the buttons in the HomeKit. I'm actually looking for a way to hide the lock button but not the device entirely. 
My motivation is to be able to monitor the doorsense and the battery level of the lock but to simply have one lock button in the HomeKit. 

Thank you!

**Comment by donavanbecker** [comment]
@Phixius5968 I could allow the door sense (contact sensor) just show by itself if you want?

**Reply by Phixius5968** [reply]
Can we tie the battery level to the contact sensor too so it stills shows in HomeKit too? 
Than would be awesome... I would also help my wife to enjoy the constant change I'm making in the home.. LOL

**Reply by donavanbecker** [reply]
yes It will be tied to the contact sensor. I will create an issue that you can follow for this feature request.

**Reply by donavanbecker** [reply]
#26
